src/components/Footer.native.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import * as pkg from '../../package.json'
+// import { TUTORIAL2_STEP_SUCCESS } from '../constants'
+import { logout, showModal } from '../action-creators'
+// import { getSetting, isTutorial } from '../selectors'
+import { scaleFontDown, scaleFontUp } from '../action-creators/scaleSize'
+import { State } from '../util/initialState'
+import { View, StyleSheet, TouchableOpacity } from 'react-native'
+import { Text } from './Text.native'
+
+/** A footer component with some useful links. */
+const Footer = () => {
+  const dispatch = useDispatch()
+  const authenticated = useSelector((state: State) => state.authenticated)
+  const user = useSelector((state: State) => state.user)
+  const status = useSelector((state: State) => state.status)
+  // const tutorialStep = useSelector((state: State) => +(getSetting(state, 'Tutorial Step') || 1))
+  // const isPushing = useSelector((state: State) => state.isPushing)
+  // const isTutorialOn = useSelector(isTutorial)
+  const pushQueueLength = useSelector((state: State) => state.pushQueue.length)
+
+  // if (isTutorialOn && tutorialStep !== TUTORIAL2_STEP_SUCCESS) return null
```

Comment:
If you switch from `mapStateToProps` to `useSelector`, please change it in both `Footer.native.tsx` and `Footer.tsx`. We want to minimize differences between the native and web components. Some differences are unavoidable, such as native components or DOM access, which we can unify in the future. But accessing the Redux state should be the same. 

You could create a hook to use Redux state for each component, e.g. `Footer.useSelectors.ts`, and share it between web and native to avoid the code duplication 🙂. This would help ensure that changes/fixes on web are propagated to native. Curious your thought on this @Foosballfan!

You are welcome to continue using `mapStateToProps` if that is easier.

Reply:
Sounds like a smart idea to me. If everyone starts converting to that convention when we touch components then we will converge on the goal sooner.

I wouldn't take a backwards step by continuing to use map state to props. Just perpetuating technical debt.

Reply:
Thanks, that feels good to me.